Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+
+RUN sudo apt-get install -y software-properties-common && sudo add-apt-repository ppa:virtuslab/git-machete && sudo apt-get update && sudo apt-get install -y python3-git-machete 
+
+
+ 
```

Comment:
Also... if you could add a step to CircleCI config, to be executed on `master` branch builds, that will automatically push the image to Dockerhub (I'll create the required image under the `gitmachete` organization)

Reply:
What's the use of that?

Reply:
Oh! is the idea for this Docker image to be built on end user's machines from Dockerfile rather than being readily downloadable from Dockerhub? 🤔

Reply:
I meant, we can push it to Dockerhub manually once, and then it will be handled by the package manager. Why push it on every change on the master if we are not building the Dockerfile from the code?

Reply:
Hmmm okay but pls note that the package you're installing here (`python3-git-machete`) is itself updated on every single `master` build: https://github.com/VirtusLab/git-machete/blob/master/.circleci/config.yml#L145

As the most recent version of `python3-git-machete` installed is 3.6.1, this effectively means that the docker image built from this Dockerfile will point to git-machete 3.6.1... at least until the new version (3.6.2? 3.7.0?) is published!

Reply:
So... I'd rather have a versioning scheme of Docker images consistent with the versions of git-machete itself 🤔

Reply:
oh ok, now I get what you mean. Let me do it.